app/controllers/api/network_routers_controller.rb

```diff
@@ -2,6 +2,16 @@ module Api
   class NetworkRoutersController < BaseController
     include Subcollections::Tags
 
+    def edit_resource(type, id, data)
+      network_router = resource_search(id, type, collection_class(:network_routers))
+      raise "Update not supported for #{network_router_ident(network_router)}" unless network_router.respond_to?(:update_network_router_queue)
```

Comment:
Do we need a supports_feature / validate_ check ?  @agrare ?

Reply:
:+1: yes `ManageIQ::Providers::Openstack::NetworkManager::NetworkRouter` has `supports :update` so we can replace this `respond_to?` with `supports_update?` and use the standard `unsupported_reason(:update)`

Reply:
This one is a little strange in that the `update_network_router_queue` doesn't live in the base `NetworkRouter` model which is why the `respond_to?` would have "worked" but supports feature is still the right way to do this